pkg/clients/gitlab.go

```diff
@@ -191,3 +192,17 @@ func IsIntEqualToIntPtr(ip *int, i int) bool {
 	}
 	return true
 }
+
+// IsStringEqualToIntPtr compares an *int with string
+func IsStringEqualToIntPtr(ip *string, i int) bool {
+	v, err := strconv.Atoi(*ip)
+	if err != nil {
+		return false
+	}
+	if ip != nil {
+		if !cmp.Equal(v, i) {
+			return false
+		}
+	}
+	return true
```

Comment:
`cmp` should only be used for complex types. There is no need to use it for simple values.

Strings and ints can be compared with `==`.